Docker build fails - need aarch64-unknown-linux-gnu

Currently the Docker Build is failing because `setup-compiler.sh` cannot complete unzipping aarch64-unknown-linux-gnu.zip from this repository.
https://github.com/dusk-network/rusk/blob/3ad0e521e21e9b949772bc65b37f663891dafb7d/scripts/setup-compiler.sh#L37

Isolated test Dockerfile

`docker build . --no-cache`


    FROM rust:latest
     
    RUN echo "Downloading compiler version v0.2.0" 
    RUN curl -L https://github.com/dusk-network/rust/releases/download/v0.2.0/aarch64-unknown-linux-gnu.zip -o "download.zip" 
    RUN unzip download.zip

```
Extracting compiler...
End-of-central-directory signature not found. Either this file is not
a zipfile, or it constitutes one disk of a multi-part archive. In the
latter case the central directory and zipfile comment will be found on
the last disk(s) of this archive.
unzip: cannot find zipfile directory in one of /opt/rusk/target/dusk/compiler/v0.2.0/duskc-aarch64-unknown-linux-gnu.zip or
/opt/rusk/target/dusk/compiler/v0.2.0/duskc-aarch64-unknown-linux-gnu.zip.zip, and cannot find /opt/rusk/target/dusk/compiler/v0.2.0/duskc-aarch64-unknown-linux-gnu.zip.ZIP, period.
make: *** [Makefile:62: setup-compiler] Error 9
```


`docker --version` Docker version 25.0.3, build 4debf41
`uname -a` :Darwin Kernel Version 23.3.0: Wed Dec 20 21:30:27 PST 2023; root:xnu-10002.81.5~7/RELEASE_ARM64_T8103 arm64 (M1 MacBook Air)

--- a/.github/workflows/dusk.yml
+++ b/.github/workflows/dusk.yml
@@ -18,14 +18,17 @@ jobs:
           x86_64-unknown-linux-gnu,
           x86_64-apple-darwin,
           aarch64-apple-darwin,
+          aarch64-unknown-linux-gnu,
         ]
         include:
-          - os: ubuntu-latest
+          - os: linux-gh-4-16-150
             target: x86_64-unknown-linux-gnu
           - os: macos-latest-large
             target: x86_64-apple-darwin
           - os: macos-latest-xlarge
             target: aarch64-apple-darwin
+          - os: arm-linux
+            target: aarch64-unknown-linux-gnu
     runs-on: ${{ matrix.os }}
     steps:
       - name: Check out repository
@@ -34,11 +37,11 @@ jobs:
           fetch-depth: 16
 
       - name: Install dependencies
-        if: matrix.os == 'ubuntu-latest'
+        if: matrix.os == 'linux-gh-4-16-150' || matrix.os == 'arm-linux'
         run: sudo apt update && sudo apt install -y ninja-build
 
       - name: Install dependencies
-        if: matrix.os == 'macos-latest'
+        if: matrix.os == 'macos-latest-large' || matrix.os == 'macos-latest-xlarge' 
         run: brew install ninja openssl@3
 
       - name: Run build

--- a/config.toml
+++ b/config.toml
@@ -20,7 +20,7 @@ target = ["wasm32-unknown-unknown", "wasm64-unknown-unknown"]
 
 # If a build of LLVM is available for the CI use it
 [llvm]
-download-ci-llvm = "if-available"
+download-ci-llvm = false
 
 [rust]
 # We have several defaults in bootstrap that depend on whether the channel

